conformance/grpc-echo/grpcecho.proto

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 The Kubernetes Authors.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "sigs.k8s.io/gateway-api/conformance/grpc-echo/grpcechoserver";
+
+package gateway_api_conformance.grpc_echo.grpcecho;
+
```

Comment:
maybe a comment here saying keeping this data type consistent with https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/echo-basic/echo-basic.go will be useful in the future

Reply:
Done.